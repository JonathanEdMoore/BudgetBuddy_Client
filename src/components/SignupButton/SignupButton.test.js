import React from 'react'
import ReactDOM from 'react-dom'
import SignupButton from './SignupButton'
import { BrowserRouter } from 'react-router-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('SignupButton smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><SignupButton /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('SignupButton snapshot test', () => {
  it('renders the UI as expected', () => {
    const wrapper = shallow(<SignupButton />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})