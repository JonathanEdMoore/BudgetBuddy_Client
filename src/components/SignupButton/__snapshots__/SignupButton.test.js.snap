// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignupButton snapshot test renders the UI as expected 1`] = `
<Link
  to="/signup"
>
  Sign Up
</Link>
`;
